src/main/target/ANYFCF7/target.h

```diff
@@ -24,6 +24,7 @@
 #define USBD_PRODUCT_STRING "AnyFCF7"
 
 #define USE_DSHOT
+#define USE_ESC_TELEMETRY
```

Comment:
@mikeller , unfortunately you can't do it in `common.h` , since `target.h` is included after `common.h` . So if you put that statement in `common.h` it would have no effect, since `USE_DSHOT` would not yet be defined.